### DISCUSSION TITLE
Does TAB conflict with Command Mode autocompletion?

### DISCUSSION THREAD

**Discussion by spencermx** [nvim-tree/nvim-tree.lua#2741] [Q&A]
Does the default TAB binding conflict with command mode autocompletion? I added a custom mapping to delete or disable that and it seems to have fixed my issue, I dont know much nvim or how keymaps work but i guess maybe i deleted the nvim-tree context TAB keymap. If I understand what happened it seems that it would be pretty much universal that everyone would want their autocompletion back and so there shouldnt be a TAB keymap to begin with.

**Comment by alex-courtis** [comment]
`<tab>` is a normal mode only mapping, see `:help nvim-tree-mappings-default`, and should not interfere with other modes.

What issue are you experiencing? Could you raise an issue with a reproducer?
